### ISSUE TITLE
404: Source code and binary

### ISSUE THREAD

**Issue by peggjt** [sanshar/Block#59]
https://sanshar.github.io/Block/build.html
The links for the source code and binary downloads are giving a 404 error.
For instance: `wget http://www.sunqm.net/pyscf/files/src/block-1.5.3.tar.gz` fails

**Comment by fishjojo** [comment]
Hi, @sunqm, do you still have the BLOCK source code and binaries available on your website? Maybe we can move them to pyscf github page? Thanks.

**Comment by sunqm** [comment]
Yes, I have backups. Would you mind to create a repo for BLOCK code and update
the relevant docs of BLOCK? I can upload the binary files to the repo.

**Comment by fishjojo** [comment]
> Yes, I have backups. Would you mind to create a repo for BLOCK code and update the relevant docs of BLOCK? I can upload the binary files to the repo.

@sunqm, I have forked the repo [here](https://github.com/pyscf/Block). I'll update the docs after you upload the binaries. Thanks.

**Comment by fishjojo** [comment]
@peggjt, the binaries can  be found [here](https://pyscf.org/Block/build#source-code-and-binary)

**Comment by peggjt** [comment]
Thank you @fishjojo

**Comment by lexming** [comment]
The binaries compiled from the source code in https://github.com/pyscf/Block still show as version 1.1.1

```
$ block.spin_adapted-serial --version
Block 1.1.1
Copyright (C) 2012  Garnet K.-L. Chan
This program comes with ABSOLUTELY NO WARRANTY; for details see license file.
This is free software, and you are welcome to redistribute it
under certain conditions; see license file for details.
```

Is the source code in that repo updated at the same level than the binaries for version 1.5.3?

**Comment by 1234zou** [comment]
I downloaded it when the link(`http://www.sunqm.net/pyscf/files/src/block-1.5.3.tar.gz`) worked some years ago.
[block-1.5.3.tar.gz](https://github.com/sanshar/Block/files/10379436/block-1.5.3.tar.gz)

**Comment by lexming** [comment]
@1234zou thanks a lot for sharing those sources. Then the sources at https://github.com/sanshar/StackBlock seem to also correspond to v1.5.3 or maybe even a bit further, as they share most of the code base with your tarball.
